core/domain/user_services.py

```diff
@@ -648,6 +626,62 @@ def _save_user_settings(user_settings):
     ).put()
 
 
+def _transform_user_settings(user_settings_model, user_id=None):
+    """Transform user settings storage model to domain object.
+
+    Args:
+        user_settings_model: UserSettingsModel.
+        user_id: str. Optional user_id to generate admin UserSettings.
```

Comment:
This seems odd. I suggest doing the admin part "outside" this function, since it's not a transformation at all in this case.

Reply:
Done.